phantomjs-env/src/main/scala/org/scalajs/jsenv/phantomjs/JettyWebsocketManager.scala

```diff
@@ -132,4 +132,12 @@ private[phantomjs] final class JettyWebsocketManager(
       webSocketConn.sendMessage(msg)
   }
 
+  def closeConnection(): Unit = {
```

Comment:
I don't fully understand why we need this. Shouldn't `stop()` do that as well?

Reply:
`stop()` is way too abrupt. It kills the connection without giving a chance to the `close` event to be triggered on the JS side. This is necessary for the normal termination to happen on the JS side.